docs/mkdocs.yml

```diff
@@ -6,7 +6,7 @@ extra_css:
   - css/style.css
   - css/algolia.css
 extra_javascript:
-  - javascript/mendablesearch.js
```

Comment:
Do we want to delete mendablesearch.js or nah?

Reply:
Good idea, done.